ocaml/xapi/certificates.ml

```diff
@@ -28,8 +28,10 @@ let c_rehash = "/usr/bin/c_rehash"
 let pem_certificate_header = "-----BEGIN CERTIFICATE-----"
 let pem_certificate_footer = "-----END CERTIFICATE-----"
 
+let certificate_path = "/etc/stunnel/certs"
```

Comment:
This could also go into `ocaml/xapi-consts/constants.ml` but I'm fine with keeping it here.